Jenkinsfile

```diff
@@ -1,49 +1,20 @@
 #!groovy
-@Library('waluigi@v5.0.0') _
-
-standardProperties()
-
-node("primary") {
-  echo "Clean workspace"
-  cleanWs()
-
-  stage("checkout") {
-    checkout localBranch(scm)
-  }
-
-  stage("dependencies") {
-    yarnInstall()
-  }
-
-  stage("stamp") {
-    sh "yarn beehive-flow stamp"
-  }
-
-  stage("build") {
-    sh "yarn build"
-  }
-
-  stage("lint") {
-    sh "yarn lint"
-  }
-
-  bedrockBrowsers()
-
-  stage("update storybook") {
-    def status = beehiveFlowStatus();
-    if (status.branchState == 'releaseReady' && status.isLatest) {
-      sshagent (credentials: ['jenkins2-github']) {
-        sh 'yarn deploy-storybook'
+@Library('waluigi@v6.0.0-rc') _
+
+beehiveFlowBuild(
```

Comment:
I did not use beehiveFlowBuild previously because the `yarn install` step does not use `--frozen-lockfile`. This would need to be fixed first.

Reply:
Ok, I see that it is different in the PR to the master branch (though oddly seems to not have been changed in the PR). Anyway, I guess the issue is not there any more.

Reply:
That's already been fixed in v6 of waluigi. It now uses the same `yarnInstall()` helper that was being used here.

Reply:
> though oddly seems to not have been changed in the PR

Yep, it was done in the previous one about setting up for containers, whereas the current active waluigi PR is just adding in more for AWS (see the feature/containers branch and my comment above about it not being fully stable yet).